circle.yml

```diff
@@ -6,6 +6,8 @@ machine:
 dependencies:
   pre:
     - ./scripts/install-dependencies.sh
+    - git lfs fetch
+    - git lfs checkout
   override:
```

Comment:
why add git lfs?

Reply:
Because the binary file is pretty large, isn't it? And it's much easier for `git` to use `lfs` for larger binary files not-line-by-line-changed... or I might be wrong, what do you think @nadade ?

Reply:
@kuba-kubula that was the idea.. but @abtris objected that git-lfs requires extra setup for the clients.. it's available by default in CircleCI and was very easy to set up on my end but that might vary for others..

Reply:
@nadade and isn't this `circle.yml` file only for CircleCI after all? cc. @abtris